cfme/tests/automate/custom_button/test_buttons.py

```diff
@@ -634,12 +633,12 @@ def test_attribute_override(appliance, request, provider, setup_provider, obj_ty
     Bugzilla:
         1651099
     """
-    attributes = [
-        ("class", "Request"),
-        ("instance", "TestNotification"),
-        ("message", "digitronik_msg"),
-        ("namespace", "/System"),
-    ]
+    attributes = {
+        "class": "Request",
+        "instance": "TestNotification",
+        "message": "digitronik_msg",
+        "namespace": "/System",
```

Comment:
**Question:**
Why aren't you using **fauxfactory** for ```message```?

Reply:
@ganeshhubale yup, fauxfactory can be better way but I'm just using old stuff; secondly its not creating something which should unique.